web/styles/components/cookieBot.ts

```diff
@@ -137,7 +137,7 @@ export const cookieBot = css`
         height: 211px;
 
         a {
-          font-size: var(--typeScale-0);
```

Comment:
<img width="362" alt="image" src="https://github.com/equinor/energyvision/assets/52911293/91623ee7-59c3-4678-8b28-313dff2d612a">
This is changing the font size at different places.

Reply:
Fixed :)